gtp2ogs.js

```diff
@@ -2225,10 +2423,21 @@ function num2gtpchar(num) { /* {{{ */
         return ".";
     return "abcdefghjklmnopqrstuvwxyz"[num];
 } /* }}} */
-function rank2str(r) { /* {{{ */
-    r = r.toFixed();
-    if (r >= 30)  return (r-30+1) + 'd';
-    else          return (30-r) + 'k';
+function rankToString(r) { /* {{{ */
+    r = Math.floor(r);
+    if (r >= 30)  return (r-30+1) + 'd'; // r>=30 : 1 dan or stronger
+    else          return (30-r) + 'k'; // r<30 : 1 kyu or weaker
+} /* }}} */
```

Comment:
Add a newline here. 

Also more general question, who started adding those random `/* }}} */` and what purpose does it serve? I think we can safely remove all of them right?

Reply:
loool not me but its quite cool, let's keep them :+1: 

for the jump line not sure where to put again, so leaving as it is too :+1: